ethsigner/subcommands/src/main/java/tech/pegasys/ethsigner/subcommands/MultiKeySubCommand.java

```diff
@@ -60,6 +60,14 @@ public MultiKeySubCommand() {}
       arity = "1")
   private Path directoryPath;
 
+  @Option(
+      names = {"-c", "--config"},
+      description = "The path to a config file to initialize signer providers",
+      paramLabel = PATH_FORMAT_HELP,
+      arity = "1",
+      required = false)
+  private Path configPath;
```

Comment:
nit: This config file update is a welcome - but it represents a second body of work ... just need to think how it fits together with doing 'signers' releases ...